entries/r/rug.nl.json

```diff
@@ -0,0 +1,16 @@
+{
+    "University of Groningen": {
+      "domain": "rug.nl",
+      "url": "https://www.rug.nl/",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```

Reply:
Alright, good to know! Will do.